examples/jupyter/search.ipynb

```diff
@@ -139,24 +135,160 @@
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/shriwise/.pyenv/versions/3.6.7/lib/python3.6/site-packages/openmc-0.10.0-py3.6-linux-x86_64.egg/openmc/surface.py:1300: FutureWarning: \"ZCylinder(...) accepts an argument named 'r', not 'R'. Future versions of OpenMC will not accept the capitalized version.\n",
+      "  warn(_WARNING_UPPER.format(type(self).__name__, 'r', 'R'), FutureWarning)\n",
```

Comment:
Can you fix this warning while we're at it?

Reply:
Can do!